### DISCUSSION TITLE
Private Key Loss for Luca-Locations

### DISCUSSION THREAD

**Discussion by stooky01** [lucaapp/security-concept#30] [Q&A]
I cannot find any infos n the web or the Luca-Locatios FAQ on how to proceed when the private key has been lost.

Any info highly appreciated.

rgds and stay healthy
stooky

**Comment by 1fabi0** [comment]
Since asymetric encryption is used the encrypted data can only be decrypted with some very high amount of computing power which still can take a few thousand years or with a not yet developed quantum computer for cryptographie

**Reply by stooky01** [reply]
yeah but that was not my question, my question is what can I do when I lose the key
I do not find any option in the app to generate a new key and start from scratch

so it looks like I have to register again whit a new mail address
